matlab_simulation/05-estimation/models/airplane_radar/airplane_radar_motion_model.m

```diff
@@ -0,0 +1,7 @@
+function M = airplane_radar_motion_model(mu, ~, dt)
+    if ~exist('dt', 'var')
```

Comment:
Maybe a single line explaining the input and output might be nice even though it might be self explanatory.